Makefile

```diff
@@ -48,6 +49,9 @@ IMAGE_TAG := latest
 DEPLOY_TAG := $(or $(DEPLOY_TAG), "")
 IMAGE_NAME=test-infra
 IMAGE_REG_NAME=quay.io/itsoiref/$(IMAGE_NAME)
+TARGET := $(or $(TARGET),minikube)
+OC_TOKEN := $(or $(OC_TOKEN),"")
+OC_SERVER := $(or $(OC_SERVER),https://api.ocp.prod.psi.redhat.com:6443)
```

Comment:
what is oc server?

Reply:
this value will be passed to oc-login command under --server flag